azure-pipelines.yml

```diff
@@ -4,11 +4,20 @@
 # https://aka.ms/yaml
 
 trigger:
-- 64097-GCP-hosting
+- 6666-automatic-gcp-deployment
```

Comment:
Will we want to set this trigger to be the main branch?

Reply:
Eventually yes. For now the best bet is probably dev, maybe we should create a pull to Main soon?

Reply:
Maybe we should have two pipelines, one for dev and one for prod? Then when we merge to dev branch the dev pipeline kicks off and pushes to our dev environment. And the prod one would just trigger off the main  branch and push to our production environment. Thoughts?

Reply:
That sounds reasonable. It is a little awkward with COTrip currently because its resources only exist in prod (as far as I know). but generally I think that makes perfect sense

Reply:
Ok, for now let's just update this to trigger off dev and push out. Next sprint we can add the production release.